### ISSUE TITLE
nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.

### ISSUE THREAD

**Issue by jchen42703** [jchen42703/reproducing-cloud-3rd-place#1]
Clogs up the prints when training
